docs/src/global/subNavigation.jsx

```diff
@@ -0,0 +1,105 @@
+import { browserHistory, withRouter } from 'react-router';
+import {
+    find,
+    flatMap,
+    kebabCase,
+    filter,
+    remove,
+} from 'lodash';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { SubNavigation as SubNav } from 'react-cm-ui';
+import { navigationItems } from '../app/navigationConstants';
+
+const propTypes = {
+    firstLevelPath: PropTypes.string.isRequired,
+    secondLevelPath: PropTypes.string.isRequired,
+    thirdLevelLabel: PropTypes.string.isRequired,
+    thirdLevelPath: PropTypes.string.isRequired,
+    router: PropTypes.shape({
+        location: PropTypes.shape({
+            pathname: PropTypes.string,
+        }),
+    }).isRequired,
+};
+
+function SubNavigation(props) {
+    const {
+        firstLevelPath,
+        secondLevelPath,
+        thirdLevelLabel,
+        thirdLevelPath,
+        router,
+    } = props;
+
+    const firstLevelItems = find(navigationItems, ['path', firstLevelPath]);
+    const secondLevelItems = find(firstLevelItems?.levelTwo, ['path', secondLevelPath]);
+    const thirdLevelItems = find(secondLevelItems?.levelThree, ['path', thirdLevelPath]);
+    const fourthLevelItems = thirdLevelItems?.levelFour;
+
+    if (!fourthLevelItems) {
+        return null;
+    }
+
+    remove(fourthLevelItems, ['emit', true]);
```

Comment:
[See here](https://github.com/saddlebackdev/react-cm-ui/pull/216#discussion_r499074788) ... might be better named as `omit` instead of `emit`.

Reply:
Fixed.

Reply:
Resolving.